src/glewlwyd.c

```diff
@@ -69,10 +69,12 @@ int main (int argc, char ** argv) {
   config->static_files_path = NULL;
   config->static_files_prefix = NULL;
   config->auth_ldap = NULL;
+  config->auth_http = NULL;
   config->refresh_token_expiration = GLEWLWYD_REFRESH_TOKEN_EXP_DEFAULT;
   config->access_token_expiration = GLEWLWYD_ACCESS_TOKEN_EXP_DEFAULT;
   config->code_expiration = GLEWLWYD_CODE_EXPIRATION_DEFAULT;
   config->jwt_decode_key = NULL;
+  config->jwt = NULL;
```

Comment:
If you do an exit_server() before the whole config structure is read in (becaus auth_http and use_scope is both on), it will stumble upon     "jwt_free((*config)->jwt);" which disappeared when I initialized config->jwt ..

Reply:
Do you imply there was a bug you fixed? (thanks :) )